app_dart/test/model/github/checks_test_data.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 /// Json messages as dart string used for checks model tests.
+library;
```

Comment:
From the docs: 

Every Dart file (plus its parts) is a [library](https://dart.dev/tools/pub/glossary#library), even if it doesn’t use a [library](https://dart.dev/language/libraries#library-directive) directive.

I've also seen some references on the web to leave this out, since it doesn't do much. Do you know if there is a recommendation from the dart team about using this keyword?

Reply:
I am wondering if that is the case then we can just ignore this.

Reply:
The recommendation from the Dart team is, if you have a Doc comment for the library, you should have a `library;` directive to put it on. If you do not have a doc comment the convention is to omit the directive. Omitting the directive with a comment is a legacy style which should be avoided.

Reply:
@natebosch thank you for the clarification on this. Much appreciated.